jbpm-wb-executor-service/jbpm-wb-executor-service-api/src/main/java/org/jbpm/workbench/es/model/events/ExecErrorSelectionEvent.java

```diff
@@ -16,22 +16,37 @@
 package org.jbpm.workbench.es.model.events;
 
 import org.jboss.errai.common.client.api.annotations.Portable;
+import org.jbpm.workbench.es.model.ExecutionErrorSummary;
 
 @Portable
 public class ExecErrorSelectionEvent {
 
     private String deploymentId;
     private String errorId;
+    private String errorDetailsTitle;
     private String serverTemplateId;
 
     public ExecErrorSelectionEvent() {
     }
 
     public ExecErrorSelectionEvent(String serverTemplateId,
+                                   ExecutionErrorSummary errorSummary
+                                   ) {
+        this(serverTemplateId,
+             errorSummary.getDeploymentId(),
+             errorSummary.getErrorId(),
+             errorSummary.getProcessId() +
```

Comment:
I think building the title String should be a responsibility of the presenter and not the event itself. I would encourage to leave the event to only hold the necessary data without much logic in it.